plugins/mcreator-core/datalists/mapcolors.yaml

```diff
@@ -1,62 +1,188 @@
-- DEFAULT
-- AIR
-- GRASS
-- SAND
-- CLOTH
-- TNT
-- ICE
-- IRON
-- FOLIAGE
-- SNOW
-- CLAY
-- DIRT
-- STONE
-- WATER
-- WOOD
-- QUARTZ
-- ADOBE
-- MAGENTA
-- LIGHT_BLUE
-- YELLOW
-- LIME
-- PINK
-- GRAY
-- SILVER
-- CYAN
-- PURPLE
-- BLUE
-- BROWN
-- GREEN
-- RED
-- BLACK
-- GOLD
-- DIAMOND
-- LAPIS
-- EMERALD
-- NETHERRACK
-- WHITE_TERRACOTTA
-- ORANGE_TERRACOTTA
-- MAGENTA_TERRACOTTA
-- LIGHT_BLUE_TERRACOTTA
-- YELLOW_TERRACOTTA
-- LIME_TERRACOTTA
-- PINK_TERRACOTTA
-- GRAY_TERRACOTTA
-- LIGHT_GRAY_TERRACOTTA
-- CYAN_TERRACOTTA
-- PURPLE_TERRACOTTA
-- BLUE_TERRACOTTA
-- BROWN_TERRACOTTA
-- GREEN_TERRACOTTA
-- RED_TERRACOTTA
-- BLACK_TERRACOTTA
-- CRIMSON_NYLIUM
-- CRIMSON_STEM
-- CRIMSON_HYPHAE
-- WARPED_NYLIUM
-- WARPED_STEM
-- WARPED_HYPHAE
-- WARPED_WART
-- DEEPSLATE_GRAY
-- RAW_IRON_PINK
-- LICHEN_GREEN
\ No newline at end of file
+- DEFAULT:
+  readable_name: "Default"
+- AIR:

```

Comment:
Iron bars for AIR/none is also strange

Reply:
I would put AIR block here instead

Reply:
When I checked the code, only iron bars and the chain blocks are still using this map color... nut if I rename the registry namr of the map color, I'll break workspaces and I didn't want to make a converter just for this.

Reply:
You can set different readable name then as "None" does not really fit if it is used by iron bars

Reply:
Since this color is "none" AIR would probably work too, as air is also not denoted on the map